ethcontract-mock/src/details/transaction.rs

```diff
@@ -2,6 +2,7 @@
 
 use ethcontract::{Address, H256, U256};
 
+#[allow(dead_code)]
```

Comment:
Got rid of the warning by removing the fields.

Reply:
I wasn't so sure I could delete it :sweat_smile: